crates/proof-of-sql/src/proof_primitive/dory/dynamic_dory_commitment_helper_gpu.rs

```diff
@@ -27,13 +45,18 @@ use tracing::{span, Level};
 #[tracing::instrument(
     name = "compute_dynamic_dory_commitments (gpu)",
     level = "debug",
-    skip_all
+    skip_all,
+    fields(total_memory, used_memory)
 )]
 pub(super) fn compute_dynamic_dory_commitments(
     committable_columns: &[CommittableColumn],
     offset: usize,
     setup: &ProverSetup,
 ) -> Vec<DynamicDoryCommitment> {
+    if tracing::level_enabled!(Level::DEBUG) {
```

Comment:
Would be more compact to put this if within `log_memory_usage()`.

Reply:
Agreed. Moved.